src/NServiceBus.Core.Tests/GlobalTestSetup.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.Core.Tests
+{
+    using System.Globalization;
+    using System.Threading;
+    using NUnit.Framework;
+
+    [SetUpFixture]
+    public class GlobalTestSetup
```

Comment:
Would you be ok if I did? Think it is a bit more discoverable

On Monday, August 29, 2016, Brandon Ording <notifications@github.com> wrote:

> In src/NServiceBus.Core.Tests/GlobalTestSetup.cs
> <https://github.com/Particular/NServiceBus/pull/4068#discussion_r76634173>
> :
>
> > @@ -0,0 +1,16 @@
> > +﻿namespace NServiceBus.Core.Tests
> > +{
> > +    using System.Globalization;
> > +    using System.Threading;
> > +    using NUnit.Framework;
> > +
> > +    [SetUpFixture]
> > +    public class GlobalTestSetup
>
> @danielmarbach <https://github.com/danielmarbach> #4068 (comment)
> <https://github.com/Particular/NServiceBus/pull/4068#discussion_r75151158>
>
> I suppose we could have renamed RedirectHelpers instead and centralized
> everything.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/4068/files/5cb6318bc289072eb682f43327cd7d28669824c6#r76634173>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKosoTaiHH65kNFHk8x6-85RIZ_OgtFks5qkwLBgaJpZM4JmLbk>
> .
>